dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -172,21 +507,167 @@ export default defineComponent({
       try {
```

Comment:
This method is really huge.
Is it possible to split it up and make it more reader-friendly?

Reply:
I will have to touch this method anyways when I will work on the new sub-tasks.

So you don't need to do the split up, since I can do it during my work on it.

Reply:
Note: Still too long.  Modularize at the end!